langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -215,3 +215,18 @@ public isolated function fromHexString(string s) returns int|error = @java:Metho
     'class: "org.ballerinalang.langlib.integer.FromHexString",
     name: "fromHexString"
 } external;
+
+# Returns the average of its arguments.
+# The result is returned as a decimal.
```

Comment:
```suggestion
```
I think the we don't need to specifically say this.

Reply:
Fixed